optimizer/plan/planbuilder.go

```diff
@@ -613,3 +615,85 @@ func (se *subqueryVisitor) Enter(in ast.Node) (out ast.Node, skipChildren bool)
 func (se *subqueryVisitor) Leave(in ast.Node) (out ast.Node, ok bool) {
 	return in, true
 }
+
+func (b *planBuilder) buildUpdate(update *ast.UpdateStmt) Plan {
+	sel := &ast.SelectStmt{From: update.TableRefs, Where: update.Where, OrderBy: update.Order, Limit: update.Limit}
```

Comment:
That is not easy to do, the mocked select statement don't have enough information to build a full select statement even we added a mocked field list.